<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>211</width>
          <height>413</height>
         </rect>
        </property>
        <property name="title">
         <string>Particula</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="10" column="0" colspan="2">
          <widget class="QPushButton" name="pbAgregaFinal">
           <property name="text">
            <string>Agregar Final</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Red</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSpinBox" name="sbRed"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="leDestinoX"/>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Ver por:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Id</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Green</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="2">
          <widget class="QPushButton" name="pbAgregarInicio">
           <property name="text">
            <string>Agregar Inicio</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QSpinBox" name="sbGreen"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="leOrigenx"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Velocidad</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="leOrigenY"/>
         </item>
         <item row="14" column="1">
          <widget class="QPushButton" name="btnOrdenarId">
           <property name="text">
            <string>Id</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>origen x</string>
           </property>
          </widget>
         </item>
         <item row="15" column="1">
          <widget class="QPushButton" name="btnOrdenDistanca">
           <property name="text">
            <string>Distancia</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Destino y</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0" colspan="2">
          <widget class="QPushButton" name="pbMostrar">
           <property name="text">
            <string>Mostrar</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="leVelocidad"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Destino x</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="leDestinoY"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>origen y</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Blue</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QSpinBox" name="sbBlue"/>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="leId"/>
         </item>
         <item row="16" column="1">
          <widget class="QPushButton" name="btnOrdenarVelocidad">
           <property name="text">
            <string>Velocidad</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPlainTextEdit" name="salida">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>10</y>
          <width>211</width>
          <height>411</height>
         </rect>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>10</y>
          <width>571</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btnDibujar">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>400</y>
          <width>291</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Dibujar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnLimpiar">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>400</y>
          <width>261</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Limpiar</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tabla</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="2">
         <widget class="QPushButton" name="btnMostrarTabla">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="lineEditTabla"/>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btnBuscar">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1053</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionGuardar_archivo"/>
    <addaction name="actionAbrir_archivo"/>
   </widget>
   <widget class="QMenu" name="menuVer">
    <property name="title">
     <string>Ver</string>
    </property>
    <addaction name="actionPuntos"/>
   </widget>
   <widget class="QMenu" name="menuAlgoritmos">
    <property name="title">
     <string>Algoritmos</string>
    </property>
    <addaction name="actionMas_cercano"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuVer"/>
   <addaction name="menuAlgoritmos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGuardar_archivo">
   <property name="text">
    <string>Guardar archivo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbrir_archivo">
   <property name="text">
    <string>Abrir archivo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionPuntos">
   <property name="text">
    <string>Puntos</string>
   </property>
  </action>
  <action name="actionMas_cercano">
   <property name="text">
    <string>Mas cercano</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
